ubuntudesign/vanilla-framework
# Add spellcheck

## Done

- Added `markdown-spellcheck` as a devDependencies
- Created a gulp task called `lint:spellcheck`
- Updated the docs to pass

## QA
- Remove node modules with `rm -rf node_modules`
- Reinstall the dependencies with `npm i`
- Run `gulp test` check there are no spelling errors
- Edit a doc to contain a spelling mistake
- Re-run `gulp test` and check you get an error

## Details

Fixes https://github.com/ubuntudesign/vanilla-framework/issues/769
